cluster-autoscaler/cloudprovider/gce/autoscaling_gce_client.go

```diff
@@ -76,6 +76,24 @@ func NewAutoscalingGceClientV1(client *http.Client, projectId string) (*autoscal
 	}, nil
 }
 
+// NewCustomAutoscalingGceClientV1 creates a new client using custom server url and timeouts
+// for communicating with GCE v1 API.
+func NewCustomAutoscalingGceClientV1(client *http.Client, projectId, serverUrl string,
```

Comment:
This is just for use in unit tests of GKE manager. Once we get rid of mocking API via http server, it'll be replaced with fake or mock.